pkg/kubelet/certificate/bootstrap/bootstrap.go

```diff
@@ -86,9 +96,16 @@ func LoadClientCert(kubeconfigPath string, bootstrapPath string, certDir string,
 	if err != nil {
 		return err
 	}
-	if err := certutil.WriteCert(certPath, certData); err != nil {
+	if err = certutil.WriteCert(certPath, certData); err != nil {
 		return err
 	}
+	cert, err := x509.ParseCertificate(certData)
+	if err != nil {
+		return err
+	}
+	if cert.NotAfter.Sub(time.Now()) <= time.Duration(0) {
```

Comment:
This has logic error? When `cert.NotAfter.Sub(time.Now()) > time.Duration(0)`, the cert will be expired.

At the same time, `time` package provides APIs to compare time, can directly use `cert.NotAfter.After(time.Now())`.